.decent_ci.yaml

```diff
@@ -3,7 +3,7 @@ results_path : _posts
 results_base_url : https://nrel.github.io/EnergyPlusBuildResults
 regression_repository : NREL/EnergyPlusRegressionTool
 regression_branch : master # this is the branch of NREL/EnergyPlusRegressionTool to use
-regression_baseline_default : develop # this is the branch to use as the baseline for regressions
+regression_baseline_default : SpaceReleaseBranchBaseline # this is the branch to use as the baseline for regressions
```

Comment:
This change just ensures that regressions will be compared the baseline branch instead of develop, which has moved since release.  This is an irrelevant change.